orka-utils.h

```diff
@@ -2,18 +2,19 @@
 #define ORKA_UTILS_H
 
 #include <stddef.h>
+#include <time.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif // __cplusplus
 
-
 extern char *
 orka_load_whole_file(const char filename[], size_t *len);
 
+long long iso8601_to_unix_ms(const char *timestamp);
 
```

Comment:
It's better to redesign this API to following the convention of scanf like this
```
int iso8601_to_unix_ms(char * timestamps,  size_t s,  void * p)

1. It returns 1 if the conversion is successfu, otherwise it returns 0
2  Inside the function 
   long long  * recipient = (long long *p);
   *recipient =   <the converted value>

this can be directly invoked by json_scanf as the following

long long  ts;
json_scanf (str, str_len, "%F",  iso8601_to_unix_ms, &ts);

```